### ISSUE TITLE
Allow constructing aiMatrix4x4t from const & pos, rot and scale

### ISSUE THREAD

**Issue by jonnenauha** [assimp/assimp#284]
Just remembered that I encountered this problem while doing the Ogre stuff. The ctor takes in refs but not const refs. My code paths failed as I was passing pos, rot and scale from const ref to a internal struct/class I had. This looked very odd to me. I looked at the code and it has nothing that would modify the passed in values. I even changed it and the build was fine after it. I reverted it before committing as I didn't want to touch anything but Ogre stuff in my pull request and thought you guys should have the final say.

The thing in question  https://github.com/assimp/assimp/blob/master/include/assimp/matrix4x4.h#L82-88

**TL;DR** These should be const :)
